R/mailR.R

```diff
@@ -186,6 +186,12 @@ send.mail <- function(from, to, subject = "", body = "", encoding = "iso-8859-1"
   if("tls" %in% names(smtp))
     if(smtp$tls)
       email$setTLS(TRUE)
+      
+  if("socketConnectionTimeout" %in% names(smtp))
+    email$setSocketConnectionTimeout(as.integer(smtp$socketConnectionTimeout));
```

Comment:
I wonder if it would make more sense (and be easier to use) if we instead include `timeout` as a parameter in `send.mail()` with a default value. What do you think?